crates/host/src/wasmbus/mod.rs

```diff
@@ -2481,14 +2480,20 @@ impl Host {
                 self.start_actor(entry, actor, actor_ref, count, host_id, annotations)
                     .await?;
             }
-            (hash_map::Entry::Occupied(entry), None) => {
-                self.stop_actor(entry, host_id).await?;
-            }
-            (hash_map::Entry::Occupied(entry), Some(count)) => {
+            (hash_map::Entry::Occupied(entry), _) => {
                 let actor = entry.get();
-                let mut instances = actor.instances.write().await;
+                let mut actor_instances = actor.instances.write().await;
                 let count = usize::from(count);
-                let current = instances.values().map(Vec::len).sum();
+                let current = actor_instances
```

Comment:
Cannot comment on an unchanged line, but this will change the value of `remaining` actors in the event(s) published, will that break any consumer usages? (this is a breaking change potentially)

Reply:
Ah I see, if an instance filter matches multiple instance lists based on annotations then it would result in multiple `actor_stopped` and `actors_stopped` events 🤔 This might require some thought, but based on your above comment I think refactoring to filter the instances by annotations and then uninstantiating all of those would be a better way to handle this.